.github/workflows/publish.yml

```
@@ -0,0 +1,82 @@
+name: Publish package
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        required: true
+        description: The release tag to fetch the package to publish
+
+  release:
+    types:
+      - published
+      - edited
+
+permissions:
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      -
+        name: Get tag version
```

Any reason why this is formatted this way ? Otherwise it would be best to remove this newline to stay consistent with the other files.
```suggestion
      - name: Get tag version
```

---

It's a try, What do you think of it ?

---

I think it decreases readability by a small amount, it doesn't seem necessary.